src/api/c/graphics_common.cpp

```diff
@@ -174,7 +174,7 @@ namespace graphics
 {
 ForgeManager& ForgeManager::getInstance()
 {
-    static ForgeManager my_instance;
+    thread_local ForgeManager my_instance;
```

Comment:
Why is graphics class thread_local ? We wanted all rendering calls queuing into a single thread, most probably main thread right ? Did the graphics example execute successfully after this change?